### PR TITLE
Update language

### PR DESCRIPTION
**Pull request by mvr320**
~~@nickygerritsen is it an idea to use the [integration](https://github.com/DOMjudge/domjudge/blob/master/gitlab/integration.sh) also to test the resulting dockers (so its clear if these packages can indeed be removed?)~~

Small typo and some cleanup, seems to work assuming that 7.2 is EOL.

### CODE REVIEW

File: docker/domserver/Dockerfile [hunk 1]
```diff
@@ -10,12 +10,8 @@ RUN apt-get update \
 	gcc g++ make acl zip unzip \
 	php-cli php-zip \
 	php-gd php-curl php-mysql php-json php-intl \
-	php-gmp php-xml php-mbstring \
 	sudo bsdmainutils ntp libcgroup-dev procps \
-	linuxdoc-tools linuxdoc-tools-text groff \
 	python3-sphinx python3-sphinx-rtd-theme python3-pip fontconfig python3-yaml \
-	texlive-latex-recommended texlive-latex-extra \
```
Comment (nickygerritsen) [inline docker/domserver/Dockerfile:17]: Don’t we need latex to build some docs still?

File: docker/domserver/Dockerfile [hunk 2]
```diff
@@ -10,12 +10,8 @@ RUN apt-get update \
 	gcc g++ make acl zip unzip \
 	php-cli php-zip \
 	php-gd php-curl php-mysql php-json php-intl \
-	php-gmp php-xml php-mbstring \
```
Comment (nickygerritsen) [inline docker/domserver/Dockerfile:13]: I think xml is/was used for the icpc cms integration but might be wrong

**Review by nickygerritsen — COMMENTED** [review]
We can do that. But what do you want to test exactly? The xml thing will be a very specific thing if still used

File: docker/domserver/Dockerfile (hunk 2 above)
Comment (mvr320) [reply docker/domserver/Dockerfile:13]: That could explain why I didn't need it when rebuilding the image.

**Comment by mvr320** [comment]
> 
> 
> We can do that. But what do you want to test exactly? The xml thing will be a very specific thing if still used

A way to know if such packages can be gone, but if you think that you know which packages are still needed it will not be that important.

File: docker/domserver/Dockerfile (hunk 1 above)
Comment (mvr320) [reply docker/domserver/Dockerfile:17]: I checked in the sources and for both `master` and `7.3` in the install texlive is not needed (but it is for `7.2`).

According to @thijskh the last `pdflatex` in the Makefiles is not needed, I checked locally and also the `groff` seems to not be needed to build the docs for master.

**Review by nickygerritsen — APPROVED** [review]

**Pull request merged**